modules/website/src/components/HomepageFeatures.js

```diff
@@ -1,62 +1,65 @@
 import React from 'react';
 import clsx from 'clsx';
 import styles from './HomepageFeatures.module.css';
+import CodeBlock from '@theme/CodeBlock';
 
 const FeatureList = [
   {
-    title: 'Contract First',
-    Svg: require('../../static/img/undraw_sync_files.svg').default,
+    title: '1. Define API Contract',
+    lang: "smithy",
+    content: "service AdminService {\n  operations: [GetAllUsers]\n}",
```

Comment:
Do you think we could cram a little bit more in there ? Like : 

```smithy
operation GetUser {
   input := { 
     @required id: String 
   } 
   output := { 
     @required firstName: String 
     @required lastName: String 
   } 
} 

service AdminService { 
  operations: [GetUser] 
} 
```

Reply:
I'll give it a go and we can see how it looks

Reply:
![screencapture-localhost-3000-smithy4s-2023-03-24-11_59_26](https://user-images.githubusercontent.com/13733772/227604187-a96ba683-ab89-482f-93a7-8ab6425f226b.png)

Reply:
Looks great, I say send it! 🚀